services/drupal/web/modules/custom/epa_workflow/epa_workflow.module

```diff
@@ -229,6 +230,28 @@ function epa_workflow_form_alter(array &$form, FormStateInterface $form_state, $
   }
 }
 
+/**
+ * Implements hook_form_BASE_FORM_ID_alter().
+ */
+function epa_workflow_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $node = $form_state->getFormObject()->getEntity();
+  if ($node instanceof NodeInterface && !$node->isNew()) {
+
+    // Loading will always return the "Current revision".
+    /** @var \Drupal\node\NodeInterface $current */
+    $current = \Drupal::entityTypeManager()
+      ->getStorage('node')
+      ->load($node->id());
+
+    // The revision ID of the "Current revision" will be lower (older) than the most recent revision.
+    if ($node->getRevisionId() > $current->getRevisionId()) {
```

Comment:
Minor quibble...but let's just check if these IDs differ at all.  It's true that the latest revision ID SHOULD always be higher than the current revision ID...but making that assumption provides us no benefit and I think we should just do a not-equals comparison here.

Reply:
Adjusted to use not-equals comparison now.